### PR TITLE
Add discovery 1/ context file

### PR DESCRIPTION
**Pull request by azaroth42**
Updating from 0/ context per discovery#92

### CODE REVIEW

**Review by mcwhitaker — DISMISSED** [review]

**Review by mixterj — DISMISSED** [review]
I created issue #2008 to account for the 'Refresh' class. I would like to have that issue taken care of before we publish a context with broken URIs referenced in it.

**Comment by azaroth42** [comment]
#2008 resolved in https://github.com/IIIF/api/pull/2006/commits/2337be18ae7efc2e369d54a6632824696c15d4e8

File: source/api/discovery/1/context.json [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+        "@type": "@vocab",
+        "@id": "dcterms:conformsTo"
+      },
+      "canonical": {
+        "@type": "@id",
+        "@id": "oa:canonical"
+      },
+      "provider": {
+        "@type": "@id",
+        "@id": "schema:provider",
```
Comment (tomcrane) [inline source/api/discovery/1/context.json:121]: schema.org not imported at top of context?

**Review by tomcrane — COMMENTED** [review]
schema:provider is used, see note

File: source/api/discovery/1/context.json (hunk 1 above)
Comment (azaroth42) [reply source/api/discovery/1/context.json:125]: Fixed, good catch!

**Review by tomcrane — APPROVED** [review]

**Review by zimeon — APPROVED** [review]

**Review by mcwhitaker — DISMISSED** [review]

**Comment by azaroth42** [comment]
Noting that this is not semantically versioned content and can be changed after publication if there are issues.

File: source/api/discovery/1/context.json [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+        "@context": {
+          "partOf": {
+            "@id": "dcterms:partOf",
+            "@type": "@id"
+          }
+        }
+      },
+
+      "Add": {
+        "@id": "as:Delete",
```
Comment (mikeapp) [inline source/api/discovery/1/context.json:70]: `as:Add` ?

File: source/api/discovery/1/context.json [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+        "@context": {
+          "partOf": {
+            "@id": "dcterms:partOf",
+            "@type": "@id"
+          }
+        }
+      },
+
+      "Remove": {
+        "@id": "as:Delete",
```
Comment (mikeapp) [inline source/api/discovery/1/context.json:80]: `as:Remove` ?

**Review by mikeapp — CHANGES_REQUESTED** [review]

**Review by mikeapp — APPROVED** [review]

**Review by kirschbombe — APPROVED** [review]

**Review by tomcrane — APPROVED** [review]

**Review by zimeon — APPROVED** [review]

**Review by mcwhitaker — APPROVED** [review]

**Pull request merged**